internal/pkg/util/configurebccsp_test.go

```diff
@@ -0,0 +1,61 @@
+// +build pkcs11
+
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
```

Comment:
2016?

Reply:
oops, I created this based off an existing file.